x/emissions/types/codec.go

```diff
@@ -3,13 +3,18 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-func RegisterCodec(_ *codec.LegacyAmino) {
+func RegisterCodec(cdc *codec.LegacyAmino) {
+	cdc.RegisterConcrete(&MsgWithdrawEmission{}, "emissions/WithdrawEmission", nil)
```

Comment:
Let's add the other message as well https://github.com/zeta-chain/node/issues/3116#issuecomment-2460214181

Reply:
Already addressed in a subsequent commit